uniffi_bindgen/src/bindings/swift/templates/Helpers.swift

```diff
@@ -53,18 +53,19 @@ fileprivate extension RustCallStatus {
 }
 
 private func rustCall<T>(_ callback: (UnsafeMutablePointer<RustCallStatus>) -> T) throws -> T {
-    try makeRustCall(callback, errorHandler: nil)
+    let neverThrow: ((RustBuffer) throws -> Never)? = nil
```

Comment:
Must do this dance so that Swift compiler knows error type, just passing a nil literal does not work.